#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("outlier_threshold", double_t, 0, "threshold to rmeove outliers", 0.01, 0.0, 0.1)
gen.add("max_iterations", int_t, 0, "the maximum iteration", 1000, 0, 10000)
gen.add("min_inliers", int_t, 0, "the minimum number on a edge", 5000, 0, 100000)
gen.add("min_points", int_t, 0, "the minimum number on a edge", 1000, 0, 100000)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "MultiPlaneSACSegmentation"))
